plugin/evm/imports_test.go

```diff
@@ -54,10 +56,12 @@ func TestMustNotImport(t *testing.T) {
 		// Importing these packages configures libevm globally and it is not
 		// possible to do so for both coreth and subnet-evm, where the client may
 		// wish to connect to multiple chains.
-		"plugin/evm/atomic": {"core", "core/vm", "params"},
-		"plugin/evm/client": {"core", "core/vm", "params"},
-		"plugin/evm/config": {"core", "core/vm", "params"},
-		"plugin/evm/header": {"core", "core/vm", "params"},
+		"plugin/evm/atomic": {"plugin/evm/customtypes", "params"},
+		"plugin/evm/client": {"plugin/evm/customtypes", "params"},
+		"plugin/evm/config": {"plugin/evm/customtypes", "params"},
+		"plugin/evm/header": {"core", "core/vm", "params"}, // Q: why do we need this?
+		"ethclient":         {"plugin/evm/customtypes", "params"},
+		"warp":              {"plugin/evm/customtypes", "params"},
```

Comment:
Why the warp addition?

Reply:
Warp has a potential customer-facing client code.